lua/telescope/builtin/internal.lua

```diff
@@ -560,15 +560,36 @@ internal.buffers = function(opts)
   }):find()
 end
 
+local colorscheme_previewer = previewers.new({
+	setup = function (self)
+		-- save original colorscheme
+		vim.api.nvim_err_writeln("Ln")
+	end,
+	preview_fn  = function(self, entry, status)
+		print("chosen entry:", entry)
+		vim.cmd("colorscheme " .. entry)
+		vim.api.nvim_err_writeln("preview_fn is not defined for this previewer")
+	end,
+	scroll_fn = function(self, dir)
```

Comment:
Will run on `actions.preview_scrolling_up`/`actions.preview_scrolling_down` is mapped to `<c-d>`/`<c-u>`